packages/flutter_tools/lib/src/observatory.dart

```diff
@@ -74,6 +74,11 @@ class Observatory {
     _getEventController(data['streamId']).add(event);
   }
 
+  void populateIsolateInfo() {
```

Comment:
Might be more obvious to see what's going on if we didn't drop the future on the floor, e.g.:

```dart
Future<Null> populateIsolateInfo() async {
  await waitFirstIsolate;
}
```